interface/__main__.py

```diff
@@ -40,5 +41,5 @@ def __run(self):
 
     Init(app=app)
     # worker = runner.init_app(app)
-    app.run(threaded=True, host="0.0.0.0", port=8000)
+    app.run(threaded=True, host="0.0.0.0", port=settings.SERVER.port)
```

Comment:
an additional field in settings.toml to receive port seems redundant, IMO.

The complete URL, with port included, is required for [registration on northstar](https://github.com/forgeflux-org/interface/blob/0c994c40132833714d4b96e92bfaac6be6f66543/interface/ns.py#L52). So if we are going to get port config separately, we'll account to account for it in ns.py as well.


An alternative would be parsing port from URL, [as done in an earlier commit](https://github.com/forgeflux-org/interface/blob/7afb328335bf051eed5a2462a91bf1c96225e2cc/interface/__main__.py#L27)

Reply:
Very well, that does seem fair as well.
Parsing the port from the URL seems like a good approach, shall I proceed to implement that into the `__main__.py` instead?